main.py

```diff
@@ -987,6 +987,12 @@ def is_integer(s):
 async def frank(ctx):
     emoji = get(bot.get_all_emojis(), name='bigfrank')
     await bot.say(emoji)
+    
+@bot.command(pass_context=True, brief="", description="")
+async def bigfrank(ctx):
+    embed = discord.Embed()
+    embed.set_image(url='https://i.imgur.com/N5bjEMl.png')
+await bot.say(embed=embed)    
```

Comment:
This is python, poor indentation will break the code base.

Tab-indent the `await bot.say(embed=embed)` line